private-templates-service/client/src/components/AdaptiveCard/AdaptiveCard.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { Provider } from "react-redux";
+import { createStore } from "redux";
+import { rootReducer } from "../../store/rootReducer";
+import AdaptiveCard from "./AdaptiveCard";
+
+it("Renders without crashing", () => {
```

Comment:
Just FYI - thorough unit tests will check if the each function you've written achieves it's purpose. 
For example, in your case, you might test whether the 'GetCard()' function returns the expected data object. 

e2e tests will come in really handy for what you're doing here - you can automate all of the 'actions' - i.e. opening a browser, clicking on a button, checking if your rendered card HTML is on the page - which is probably the most accurate way of testing this.

But for now, I wouldn't change anything - I think it's a little too early to be worrying about robust unit tests.

Reply:
Okay, that makes sense. I will add a ticket to our backlog to add more robust unit tests once we have a more base.